src/tracks/playabletrack/wavetrack/WaveTrackUtils.cpp

```diff
@@ -11,10 +11,23 @@
 #include "WaveTrackUtils.h"
 #include "ViewInfo.h"
 #include "WaveClip.h"
-
+#include "WaveTrack.h"
 
 bool WaveTrackUtils::IsClipSelected(const ViewInfo& viewInfo, const WaveClip& clip)
 {
    return clip.GetPlayStartTime() == viewInfo.selectedRegion.t0() &&
       clip.GetPlayEndTime() == viewInfo.selectedRegion.t1();
 }
+
+TranslatableString
+WaveTrackUtils::GetChannelUnlinkReason(WaveTrack& left, WaveTrack& right)
+{
+   if(left.GetRate() != right.GetRate())
+      //i18n-hint: explains why opened project was auto-modified 
+      return XO("This project contained stereo tracks with different sample rates per channel.");
+   if(left.GetSampleFormat() != right.GetSampleFormat())
+      //i18n-hint: explains why opened project was auto-modified  
+      return XO("This project contained stereo tracks with different sample formats in channels.");
+   //i18n-hint: explains why opened project was auto-modified 
+   return XO("This project contained stereo tracks with non-aligned content.");
```

Comment:
Strings, @LWinterberg  ?

This function causes only the first applicable message to be shown.  Possibly more that one might apply in an old project.  Maybe all applicable messages should be shown.  Or maybe we don't care about this edge case of an edge case.

Reply:
strings look good!